app/views/root/designprinciples.html.erb

```diff
@@ -44,7 +44,8 @@
      <li class="principle">
       <article>
         <hgroup>
-          <h1 id="first">Start with user needs</h1>
+          <h1 id="first">Start with user needs*</h1>
+          <h2>*user needs not government needs</h2>
```

Comment:
User needs is now duplicated and feels redundant IMO. I suggest **IF** we think that it's important to have 'not government needs' in there that we just make it part of the H1, for example:

> **Start with user needs, not government needs**

It may of course make sense to use something other than a comma to punctuate…

We should then remove the `<hgroup>` which is also redundant.